coprocessor/fhevm-engine/tfhe-worker/benches/utils.rs

```diff
@@ -108,6 +108,7 @@ async fn start_coprocessor(rx: Receiver<bool>, app_port: u16, db_url: &str) {
         health_check_port: 8080,
         metric_rerand_batch_latency: MetricsConfig::default(),
         metric_fhe_batch_latency: MetricsConfig::default(),
+        worker_id: None,
```

Comment:
Couldn't they invent a unique id on start or from a table ? If we have dozen workers, that could be error prone.

Reply:
That's optional. If `worker_id` is not provided, the worker generates its own uuid.